src/actions/selection.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+export default function selectionReducer (state, action) {
+	if (action.type === 'hermesChangeSelection') {
```

Comment:
I didn't have the `hermes` prefix on the other action names, maybe this isn't necessary, investigate.